### DISCUSSION TITLE
Can't parse custom design params and Translation error

### DISCUSSION THREAD

**Discussion by oksuzgil** [mynaparrot/plugNmeet-server#178] [Q&A] [answered]
Plugnmeet domain = plugnmeet.example.com
Css domain = example.com

` window.DESIGN_CUSTOMIZATION = `{
   "primary_color": "#004D90",
   "secondary_color": "#24AEF7",
   "background_color": "#0b7db4",
   "background_image": "https:\/\/example.com\/video\/custom_bg.png",
   "header_bg_color": "#45b3ec",
   "footer_bg_color": "#45b3ec",
   "left_side_bg_color": "#04a2f3",
   "right_side_bg_color": "#04a2f3",
   "custom_css_url": "https:\/\/example.com\/video\/plugNmeet_desing.css",
   "custom_logo": "https:\/\/example.com\/video\/logo.png",
 }`;
`

but I am getting "Can't parse custom design params" error.

Also, the error "https://plugnmeet.example.com/assets/locales/tr/translation.json 404 (Not Found)" is returned as a language file. When I check on the server side, the language files are found as "/client/dist/assets/locales/tr-TR/translation.json".

**Comment by jibon57** [comment, marked as answer]
You can ignore translation file missing. Make sure that json code is valid. You can validated from here: https://jsonlint.com/
